prow/cmd/line/main.go

```diff
@@ -257,7 +260,15 @@ func (c *testClient) TestPRKubernetes() error {
 // TestPRJenkins starts a Jenkins build and watches it, updating the GitHub
 // status as necessary.
 func (c *testClient) TestPRJenkins() error {
+	if size, err := c.JenkinsClient.QueueSize(); err != nil {
```

Comment:
it seems bad that a (transient?) 500 getting the queue size mean that you won't even try to start the build